### ISSUE TITLE
First try at Cython

### ISSUE THREAD

**Issue by MaxHalford** [creme-ml/creme#5]
Most of the time `creme` is slower than `sklearn`. This is mostly due to the fact that `sklearn` can vectorize code because all the data is available in memory at once. There might be some low hanging fruits that we can optimize using Cython instead of pure Python. A good start might be "cythonize" some of the optimizers in the `optim` module and see if there is a noticeable gain.

**Comment by MaxHalford** [comment]
Done! `Mean`, `Var`, and the loss functions are now written in Cython.
